### PR TITLE
[SOAR-9386 ] SQL - Fix .get method

### PR DESCRIPTION
**Pull request by igorski-r7**
### Description

Describe the proposed changes:

  - Fixed .get method keyword argument error

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [X] Screenshot of job output with the plugin changes
- [X] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [X] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [X] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [X] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [X] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [X] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [X] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [X] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [X] Work fully completed
- [X] Functional
  - [X] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [X] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [X] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [X] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [X] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### Run

<details>

```
{
  "body": {
    "log": "Using PostgreSQL connection string...\nrapid7/https://example.com Step name: query\n{'Node Type': 'Seq Scan', 'Parallel Aware': False, 'Async Capable': False, 'Relation Name': 'firewalls_config', 'Alias': 'firewalls_config', 'Startup Cost': https://example.com, 'Total Cost': https://example.com, 'Plan Rows': 2550, 'Plan Width': 4, 'Actual Startup Time': https://example.com, 'Actual Total Time': https://example.com, 'Actual Rows': 1, 'Actual Loops': 1, 'Shared Hit Blocks': 1, 'Shared Read Blocks': 0, 'Shared Dirtied Blocks': 0, 'Shared Written Blocks': 0, 'Local Hit Blocks': 0, 'Local Read Blocks': 0, 'Local Dirtied Blocks': 0, 'Local Written Blocks': 0, 'Temp Read Blocks': 0, 'Temp Written Blocks': 0}\n",
    "meta": {},
    "output": {
      "header": [
        "number_test"
      ],
      "results": [
        {
          "number_test": "123"
        }
      ],
      "status": "operation success"
    },
    "status": "ok"
  },
  "type": "action_event",
  "version": "v1"
}

```

<summary>
docker run --rm -i rapid7/sql:3.0.4 run < tests/query.json
</summary>
</details>

### CODE REVIEW

**Review by ablakley-r7 — COMMENTED** [review]
Change looks great but noticed unit test was failing due to komand import. Is updating to insightconnect_plugin_runtime part of the remit of the ticket or does this test pass locally?

**Comment by igorski-r7** [comment]
> Change looks great but noticed unit test was failing due to komand import. Is updating to insightconnect_plugin_runtime part of the remit of the ticket or does this test pass locally?

It fails the same way locally as well. Well, updating that plugin to insightconnect_plugin_runtime is not a part of that ticket. My suggested solution to that check is to delete that unittests (because there is only one test).

File: plugins/sql/help.md [hunk 1]
```diff
@@ -119,6 +119,7 @@ For the SQL query action, be sure that your query is valid SQL.
 
 # Version History
 
+* 3.0.4 - Fix .get keyword argument error
```
Comment (mberezin-r7) [inline plugins/sql/help.md:122]:
```suggestion
* 3.0.4 - Patch keyword argument error
```

**Review by mberezin-r7 — COMMENTED** [review]
Looks good! Only recommendation is I would simplify the version history message :)

**Review by mberezin-r7 — APPROVED** [review]

**Review by ablakley-r7 — APPROVED** [review]

**Pull request closed without merging**